UnityProject/Assets/Scripts/UI/Systems/EndRound/RoundEndScoreScreen.cs

```diff
@@ -1,10 +1,12 @@
 using System;
 using System.Collections.Generic;
+using System.Linq;
 using System.Text;
 using Messages.Server;
 using Mirror;
 using Systems.Score;
 using TMPro;
+using UnityEditor;
```

Comment:
this namespace can't be used in runtime. I don't see you using it anywhere, so you can either delete it or wrap in a ``# if UNITY_EDITOR`` preprocessor

Reply:
This namespace shouldn't be even here, deleted.